tests/jballerina-unit-test/src/test/resources/test-src/expressions/rawtemplate/raw_template_literal_test.bal

```diff
@@ -203,19 +203,20 @@ function testUseWithVar() {
     var rt = `Hello ${name}!`;
     typedesc<any> td = typeof rt;
 
-    assert("typedesc $anonType$21 {\n\tstrings : string[],\n\tinsertions : (any|error)[]\n}", td.toString());
+    assert("typedesc $rawTemplate$RawTemplate$12",
```

Comment:
Do we have tests that assert the strings and insertions? Asserting the typedesc doesn't quite validate behaviour IMO.

Reply:
Yeah, those are the normal tests. This one is just to test if a type was desugared for this properly.